test/studies/hpcc/RA/marybeth/ra2-param-test.chpl

```diff
@@ -130,7 +130,7 @@ proc VerifyResults() {
   }
 }
 
-proc Timer():real {
+proc stopwatch():real {
```

Comment:
Maybe not related to this PR, but it seems odd that this proc just returns 1, but it's being used as a Timer/stopwatch above...

Reply:
I think the idea was for it to fake being a timer to simplify the test. So it wasn't meant to be a benchmark, but just to look somewhat like one.